client/src/components/QandA/QuestionsList/Answers.jsx

```diff
@@ -21,7 +21,7 @@ const Answers = ({ answers }) => {
 
   return (
     <>
-      {sortedAnswers.map(ans =>
+      {sortedAnswers.slice(0, answerCounter).map(ans =>
         <div>
```

Comment:
i think if you move the key attribute into the div here the error message in console will go away.

Reply:
Oh ok! I'll try that and see if the pop-up stops.